ui/pages/confirmation/templates/add-ethereum-chain.js

```diff
@@ -340,6 +367,7 @@ function getValues(pendingApproval, t, actions, history) {
         actions.addCustomNetwork(pendingApproval.requestData);
         history.push(DEFAULT_ROUTE);
       }
+      return {};
```

Comment:
Nit: Perhaps better to return `undefined` if the return value is not expected to be used

```suggestion
      return undefined;
```